spec/components/check_box_action_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+RSpec.describe TaskList::CheckBoxAction, type: :component do
+  let(:task) { double(status: :not_started, locales_path: "conversion.voluntary.tasks.handover") }
+
+  before do
+    allow(task).to receive(attribute.to_sym).and_return(true)
+  end
+
+  context "when the checkbox only has a title" do
+    let(:attribute) { "notes" }
+
+    it "renders the title" do
+      render_inline(TaskList::CheckBoxAction.new(task: task, attribute: attribute))
```

Comment:
This `render_inline` could be placed in the before block and removed from each test, but that's probably just personal preference.